azure-pipelines.yml

```
@@ -6,6 +6,9 @@ strategy:
     postgres:
       imageName: 'postgres'
       version: '11.9'
+    postgres:
```

May be this (if pipeline works) ?

```suggestion
    postgres11_3:
```